### PR TITLE
Add kas-container 3.0.2 to install hooks

### PR DESCRIPTION
**Pull request by tym2k1**

### CODE REVIEW

**Review by Kewkaa — COMMENTED** [review]
Za pierwszym razem po wykonaniu komendy:
`ansible-playbook --ask-become-pass --extra-vars @extra_vars/$USER.yml workstation-setup-eft.yml`
wykonywanie się zawiesiło, musiałam anulować crtrl+c. Za drugim razem poszło be problemów. zsh zrestartowany, kas-container działa dobrze.

**Comment by macpijan** [comment]
@Kewkaa We shall use English in public repositories.

If you are happy with the MR, you can leave a review and approve it.

**Review by Kewkaa — APPROVED** [review]
The first time after executing the command:
`ansible-playbook --ask-become-pass --extra-vars @extra_vars/$USER.yml workstation-setup-eft.yml`
The execution crashed, I had to cancel crtrl+c. Second time went without problems. zsh restarted, kas-container works fine.

**Review by PLangowski — APPROVED** [review]
This works

File: roles/ohmyzsh/templates/zshrc.j2 [hunk 1]
```diff
@@ -98,4 +98,11 @@ if [ -e ~/.nix-profile/etc/profile.d/nix.sh ]; then
   source ~/.nix-profile/etc/profile.d/nix.sh
 fi
 
+# User specific environment
+if ! [[ "$PATH" =~ "$HOME/.local/bin:$HOME/bin:" ]]
+then
+    PATH="$HOME/.local/bin:$HOME/bin:$PATH"
+fi
+export PATH
```
[the remark this answers is not in the document]
Comment (tym2k1) [reply roles/ohmyzsh/templates/zshrc.j2:106]: Already solved by d8ddf0a9503e32ed1336577de5d42940f6133405

**Comment by DaniilKl** [comment]
It works, indeed

**Review by DaniilKl — APPROVED** [review]
It works, indeed

**Comment by tym2k1** [comment]
@macpijan I also think it's more elegant that way. Now kas-container is installed in /.local/bin, and /bin is no longer added automatically to $PATH.

**Review by TomaszAIR — APPROVED** [review]

**Pull request merged**